frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,32 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce'
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce((event) => {
+        debugger;
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
+            return;
+        }
+
+        ajax({
+            url: checkerURL + '?email=' + emailField.value,
```

Comment:
I was thinking more about the fact that special characters in the email address with mess with the request URL. For example, an email address with an `&` in it will screw up the query params.